### ISSUE TITLE
Annex H (SPDX File Tags), but for snippets

### ISSUE THREAD

**Issue by Artoria2e5** [spdx/spdx-spec#852]
There may be a point in extending Annex H to cover snippets, so these concatenated javascript files can have a machine-readable licensing information for each range. (Having markers embedded into the file is more robust to edits, but then the file is probably not auto-generated and I am running out of examples.)

Now the question is to decide how to mark snippets. If we are okay with making `SnippetSPDXID`  mandatory, we can say that each snippet starts with a `SPDX-SnippetSPDXID:` and ends with the next `SPDX-SnippetSPDXID`. It would be good to have something that ends a snippet without beginning one.

There's also the question of actually marking the license. We can either use the snippet properties (`SPDX-LicenseInfoInSnippet:` and friends), or just say that Annex E `SPDX-License-Identifier:` is scoped.

**Comment by Freso** [comment]
Seems like this was solved in https://github.com/spdx/spdx-spec/pull/719 — but I can’t find anything akin to annex H in v3.0.1.

**Comment by Artoria2e5** [comment]
Yep, no hits on `SPDX-License-Identifier` in any markdown file in this repo either...

**Comment by Freso** [comment]
I opened a separate issue about their current status: https://github.com/spdx/spdx-spec/issues/1393

**Comment by zvr** [comment]
Can we close both of these ticket?

I've provided the answer on a [comment](https://github.com/spdx/spdx-spec/issues/1393#issuecomment-4501684683) no the other one.
